### ISSUE TITLE
You should prefer using authentication method using verification link

### ISSUE THREAD

**Issue by saurabhcoded** [dhruv9316/StudyNotion-Mega-Project#3]
You are using old verification method based on otp, prefer using verification link sending.

1. Once a user register directly let them access their Dashboard.
2. Unless they verify their email address they can't purchase courses or create new courses.
3. This will allow your project to people can use for their own purpose.

**Comment by dhruv9316** [comment]
OK I understand your opinion , but I think this will be hard to change the
whole authentication process as some other processes are also dependent on
that, secondly i dont know why i didnt find this issue while registering in
atleast 5 gmails. Is it the problem related to scalability do u know
something about that.

**Comment by dhruv9316** [comment]
hey bro I am recognising this now that otp verification is working fine in
pc but not in mobile, do you know something about this

**Comment by saurabhcoded** [comment]
Yes you won't be able to make a scalable structure in this kind of authentication system. you can contact with me for any guidance https://www.linkedin.com/in/saurabhcoded/ and will be happy to help you or contribute to your project.
